integrationTest/integration_test.go

```diff
@@ -1594,6 +1598,12 @@ func makeTests() []*TestGroup {
 			tc("Modify DHCID record", dhcid("test", "Test/AuCccgoJbsaxcQc9TUapptP69lOjxfNuVAA2kjEA=")),
 		),
 
+		testgroup("DNAME",
+			requires(providers.CanUseDNAME),
+			tc("Create DNAME record", dname("test", "example.com.")),
+			tc("Modify DNAME record", dname("test", "example.net.")),
```

Comment:
Please also add a case where you add a non-FQDN name.  For example:

```
               testgroup("DNAME-short",
                       tc("Create a DNAME",
                               a("short", "1.2.3.4"),
                               dname("testdname", "short"),
                       ),
```

(I'm not sure if the `a()` is required)